bool    save
string  storage_path
string  name
---
bool    success
crop_image_processing/Crop crop
